src/libraries/System.Data.Common/src/System/Data/DataColumn.cs

```diff
@@ -76,6 +78,8 @@ public class DataColumn : MarshalByValueComponent
         /// Initializes a new instance of a <see cref='System.Data.DataColumn'/>
         /// class.
         /// </summary>
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
```

Comment:
NIT: for "IL2026" we usually just add RequiresUnreferencedCode, instead of UnrecognizedReflectionPattern